### PR TITLE
en: refine quick-start docs

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

fix issue pingcap/dm#782 and pingcap/dm#781

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/151 and https://github.com/pingcap/docs-dm/pull/190
- Other reference link(s):

### CODE REVIEW

File: en/quick-start-create-task.md [hunk 1]
```diff
@@ -1,39 +1,72 @@
 ---
-title: Create and Verify a Data Replication Task
-summary: Learn how to create a replication task to verify whether the cluster works well after the DM cluster is deployed.
-aliases: ['/docs/tidb-data-migration/dev/create-task-and-verify/']
+title: Create a Data Migration Task
+summary: Learn how to create a migration task after the DM cluster is deployed.
+aliases: ['/zh/tidb-data-migration/dev/create-task-and-verify']
```
Comment (csuzhangxc) [inline en/quick-start-create-task.md:4]: `zh`?

File: en/quick-start-create-task.md [hunk 2]
```diff
@@ -1,39 +1,72 @@
...
+docker run --rm --name mysql-3307 -p 3307:3307 -e MYSQL_ALLOW_EMPTY_PASSWORD=true mysql:5.7.22 --log-bin=mysql-bin --port=3307 --bind-address=0.0.0.0 --binlog-format=ROW --server-id=1 --gtid_mode=ON --enforce-gtid-consistency=true > mysql.3307.log 2>&1 &
+```
+
+### Prepare data
+
+- Write [example data](https://github.com/pingcap/dm/blob/bc1094a6b7388ad934279898b4e308cd3d58f7a9/tests/sharding/data/db1.prepare.sql) into mysql-3306.
+
+- Write [example data](https://github.com/pingcap/dm/blob/bc1094a6b7388ad934279898b4e308cd3d58f7a9/tests/sharding/data/db2.prepare.sql) into mysql-3307. 
+
+### Start downstream MySQL
```
Comment (csuzhangxc) [inline en/quick-start-create-task.md:46]:
```suggestion
### Start downstream TiDB
```

File: en/quick-start-create-task.md [hunk 3]
```diff
@@ -47,26 +80,30 @@ fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg=
...
+
+from:
+  host: "127.0.0.1"
+  user: "root"
+  password: "fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg="
+  port: 3306
 ```
 
-In MySQL2 data source, copy the above configurations to `conf/source2.toml`. You need to change `name` to `mysql-replica-02`, `host` to `192.168.0.2`, and change `password` and `port` to appropriate values.
+In MySQL2 data source, copy the above configurations to `conf/source2.yaml`. You need to change `password` and `port` to appropriate values.
```
Comment (csuzhangxc) [inline en/quick-start-create-task.md:106]: if you also translated #190, please udpate `source.toml` in `en/manger-source.md` to `source.yaml`.

File: en/replicate-data-using-dm.md [hunk 4]
```diff
@@ -57,7 +57,35 @@ After the DM cluster is deployed using DM-Ansible, the configuration information
     >
     > The `{ansible deploy}` in `{ansible deploy}/conf/dm-master.toml` indicates the directory where DM-Ansible is deployed. It is the directory configured in the `deploy_dir` parameter.
 
-## Step 3: Configure the data replication task
+## Step 3: Create data source
+
+1. Write MySQL-1 related information to `conf/source1.toml`:
```
Comment (csuzhangxc) [inline en/replicate-data-using-dm.md:62]: choose `source.toml` ((before #190) or `source.yaml` (after #190) in this PR.

File: en/quick-start-create-task.md (hunk 3 above)
Comment (ireneontheway) [reply en/quick-start-create-task.md:106]: Yes, I am doing this PR as well.

**Comment by ireneontheway** [comment]
PTAL @yikeke @csuzhangxc

File: en/manage-source.md [hunk 5]
```diff
@@ -52,6 +52,8 @@ Global Flags:
 operate-source create ./source.toml
 ```
 
+For the configuration of `source.toml`, refer to [Upstream Database Configuration File Introduction](source-configuration-file.md).
```
Comment (csuzhangxc) [inline en/manage-source.md:55]:
```suggestion
For the configuration of `source.yaml`, refer to [Upstream Database Configuration File Introduction](source-configuration-file.md).
```

**Comment by ireneontheway** [comment]
@yikeke PTAL

File: en/quick-start-create-task.md [hunk 6]
```diff
@@ -1,39 +1,72 @@
 ---
-title: Create and Verify a Data Replication Task
-summary: Learn how to create a replication task to verify whether the cluster works well after the DM cluster is deployed.
-aliases: ['/docs/tidb-data-migration/dev/create-task-and-verify/']
+title: Create a Data Migration Task
+summary: Learn how to create a migration task after the DM cluster is deployed.
+aliases: ['/en/tidb-data-migration/dev/create-task-and-verify']
```
Comment (yikeke) [inline en/quick-start-create-task.md:4]:
```suggestion
aliases: ['/docs/tidb-data-migration/dev/create-task-and-verify/','/tidb-data-migration/dev/create-task-and-verify']
```
Warning: Do not change the existing aliases (they exist for a reason!). @ireneontheway 

Here you need to add a new alias.

File: en/quick-start-create-task.md [hunk 7]
```diff
@@ -47,26 +80,30 @@ fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg=
 
 Save this encrypted value, and use it for creating a MySQL data source in the following steps.
 
+> **Warning:**
```
Comment (yikeke) [inline en/quick-start-create-task.md:83]:
```suggestion
> **Note:**
```

File: en/quick-start-create-task.md (hunk 3 above)
Comment (yikeke) [inline en/quick-start-create-task.md:106]:
```suggestion
In MySQL2 data source, copy the above configurations to `conf/source2.yaml`. You need to change `name` to `mysql-replica-02` and change `password` and `port` to appropriate values.
```

File: en/quick-start-create-task.md [hunk 8]
```diff
@@ -75,14 +112,14 @@ To load the data source configurations of MySQL1 into the DM cluster using dmctl
...
+./bin/dmctl --master-addr=127.0.0.1:8261 operate-source create conf/source1.yaml
 ```
 
 For MySQL2, replace the configuration file in the above command with that of MySQL2.
 
-## Create a data replication task
+## Create a data migration task
 
-Suppose that there are several sharded tables on both MySQL1 and MySQL2 instances. These tables have identical structure and the same prefix “t” in the table names; the databases where these tables are located are all prefixed with "sharding"; and there is no conflict between the primary keys or the unique keys (in each sharded table, the primary keys or the unique keys are different from those of other tables). 
+After importing [prepare data](#Prepare data),there are several sharded tables on both MySQL1 and MySQL2 instances. These tables have identical structure and the same prefix “t” in the table names; the databases where these tables are located are all prefixed with "sharding"; and there is no conflict between the primary keys or the unique keys (in each sharded table, the primary keys or the unique keys are different from those of other tables).
```
Comment (yikeke) [inline en/quick-start-create-task.md:122]:
```suggestion
After importing [prepared data](#prepare-data), there are several sharded tables on both MySQL1 and MySQL2 instances. These tables have identical structure and the same prefix “t” in the table names; the databases where these tables are located are all prefixed with "sharding"; and there is no conflict between the primary keys or the unique keys (in each sharded table, the primary keys or the unique keys are different from those of other tables). 
```
Please read through this [link](https://docs.google.com/document/d/1b6ZhZD33OoM8AacpKksGGSuxJWReLkNnSt8eSc1kTXc/edit?ts=5c9c682e#heading=h.xy6k1sq6eh08) section in style guide and use the link anchors correctly. @ireneontheway

**Comment by ireneontheway** [comment]
This PR combined the translation of https://github.com/pingcap/docs-dm/pull/190 and https://github.com/pingcap/docs-dm/pull/151 because both PRs have replaced `toml` into `yaml`. @yikeke @csuzhangxc

File: en/quick-start-create-task.md [hunk 9]
```diff
@@ -47,26 +80,30 @@ fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg=
...
 
 # Indicates whether GTID is enabled
-enable-gtid = false
- 
-[from]
-host = "192.168.0.1"
-user = "root"
-password = "fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg="
-port = 3306
+enable-gtid = true
```
Comment (yikeke) [inline en/quick-start-create-task.md:97]:
```suggestion
enable-gtid: true
```
to align https://github.com/pingcap/docs-dm/pull/190/files#diff-26ed429c15c1301384a8b755d75015c5R94

File: en/replicate-data-using-dm.md [hunk 10]
```diff
@@ -57,7 +57,35 @@ After the DM cluster is deployed using DM-Ansible, the configuration information
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    ./bin/dmctl --master-addr=127.0.0.1:8261 operate-source create conf/source1.yaml
+    ```
+
+3. For MySQL-2, modify the relevant information in the configuration file and execute the same `dmctl` command.
+
+## Step 4: Configure the data replication task
```
Comment (yikeke) [inline en/replicate-data-using-dm.md:88]: It's weird that "data replication task" is used in en/replicate-data-using-dm.md and "data migration task" is used in en/quick-start-create-task.md. The two documents are actually in the same context. As confirmed with @csuzhangxc , please help update all "data replication task" to "data migration task" in this document. @ireneontheway

File: en/source-configuration-file.md [hunk 11]
```diff
@@ -12,17 +12,17 @@ This document introduces the configuration file of the upstream database, includ
...
-[from]
-host = "127.0.0.1"
-port = 3306
-user = "root"
-password = "ZqMLjZ2j5khNelDEfDoUhkD5aV5fIJOe0fiog9w=" # The user password of the upstream database. Note that the password must be encrypted using dmctl.
+from:
+host: "127.0.0.1"
+port: 3306
+user: "root"
+password: "ZqMLjZ2j5khNelDEfDoUhkD5aV5fIJOe0fiog9w=" # The user password of the upstream database. Note that the password must be encrypted using dmctl.
```
Comment (csuzhangxc) [inline en/source-configuration-file.md:25]: It should be two spaces (`  `) before these lines as we did in #190.

Comment (ireneontheway) [reply en/source-configuration-file.md]: Done

**Pull request merged**